docs/docs/11_deploy_to_render/06_run_everything_docker_compose/start/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# CONTRIBUTING
+
+## How to run the Dockerfile locally
+
+```
+docker run -dp 5000:5000 -w /app -v "$(pwd):/app" IMAGE_NAME sh -c "flask run"
```

Comment:
Add example commands for Windows ([see examples here](https://github.com/tecladocode/rest-apis-flask-python/pull/147/commits/bb61c5350be7c3fb666d9bcf9f08a8409c60fa5b#r1551545870).